Standards/CCSMB-5.md

```diff
@@ -7,17 +7,17 @@
 A client will only listen to packets while the audio is being transmitted from a server
 The server will communicate to the client over modem
 
+
 ## Packet
 A packet consists of the following fields in a table:
-- buffer: A table that contains Signed 8-bit PCM @ 48kHz audio
-- station: A string that contains the name of the radio station
-- title: A string that contains the song that is currently being played
-- protocol: A string that contains the protocol used
+- `buffer`: A table that contains Signed 8-bit PCM @ 48kHz audio
+- `id`: The ID of the server sending the message
+- `station`: A string that contains the name of the radio station
+- `title`: A string that contains the song that is currently being played, or other relevant metadata
```

Comment:
```suggestion
`title`: A string that contains the song that is currently being played
`metadata`: A string that contains some relevant metadata (optional)
```
I would like to suggest putting metadata in a separate field. In my opinion, it is not worth putting this information in the "Title" field.
Let's say we have a public radio receiver that shows the current song. It is unlikely that users want to see there, for example, the name of album from which this song is taken.

Reply:
I think this is a good idea, waiting for response of others first though

Reply:
Hmm that could work as well, but would limit the metadata that can be sent. just `metadata` seems quite permissive and undefined

Reply:
i feel like metadata should be a table
but that is just my preference

Reply:
We'd need to define set fields within the metadata table to avoid ambiguity, but that's possible